pkg/cincinnati/cincinnati.go

```diff
@@ -177,16 +177,36 @@ func CalculateUpgrades(ctx context.Context, c Client, arch, sourceChannel, targe
 		return GetUpdates(ctx, c, arch, targetChannel, startVer, reqVer)
 	}
 
+	// Check the major and minor versions are the same with different
+	// channel prefixes
+	source, target, _, err := getSemverFromChannels(sourceChannel, targetChannel)
+	if err != nil {
+		return Update{}, Update{}, nil, err
```

Comment:
Should it be `return source, target, nil, err` instead?

Reply:
`source` and `target` are the `semver.Version` representation of the source and target channel suffixes (e.g. source could be 4.9.0 and target could be 4.10.0). So in this case no, but maybe a variable rename or comment could make that more clear.